spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -751,15 +751,14 @@ An `UnsupportedOperationException` is thrown if the active context object for th
 
 ==== The built-in `Instance`
 
-The container must provide a built-in bean with:
+For each injection point of `Instance<?>` or `Provider<?>`, the container must provide a built-in bean that satisfies:
 
 * `Instance<X>` and `Provider<X>` for every legal bean type `X` in its set of bean types,
```

Comment:
CDI-232 is about this sentence that needs to be changed. 

I propose something like:

The container must provide a built-in bean:
* which is eligible for injection to any injection point with required type `Instance<X>` or `Provider<X>`, where `X` is any legal bean type, and any required qualifiers
* has `@Dependent` scope,
* has no bean name,
* an implementation is provided automatically by the container.